qdrant/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bee8e975-219c-4673-83c0-9fa68b973298",
+  "app_id": "qdrant",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "A high-performance vector search engine/database.",
+    "title": "Qdrant",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "A Qdrant Integration Dashboard for Datadog",
+        "image_url": "images/qdrant_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::",
+      "Offering::",
+      "Queried Data Type::",
+      "Submitted Data Type::"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id": 18715225,
+      "source_type_name": "Qdrant",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "qdrant.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "qdrant-overview": "assets/dashboards/qdrant_overview.json"
+    },
+    "monitors": {},
+    "logs": {},
```

Comment:
if you don't have monitors and logs remove them from this file.

Reply:
[`669e46f` (#2422)](https://github.com/DataDog/integrations-extras/pull/2422/commits/669e46fd4ca6af5ce106c2650106a3c61f65017a)